draft-ietf-quic-manageability.md

```diff
@@ -405,7 +405,8 @@ information; observing it serves only to determine that the handshake has
 completed.
```

Comment:
I'm actually not sure how one determines the handshake is complete from the server completion.  How does one tell the difference between the server sending 0.5RTT and 1RTT containing HANDSHAKE_DONE?

Reply:
Maybe this is worth a note:
> The Server Completion flight might not be observable as it is indistinguishable from packets the server sends after the Server Initial flight.